content/blog/hackhaton-2022-building-a-pong.md

```diff
@@ -0,0 +1,213 @@
+
+---
+Title: Hackapong Fun at the Shop!
+Date: 2022-11-04 13:55:00
+Author: Jean-Philippe Lambert & Alex Hoguin
+Category: Hackathon
+Tags: hackathon, dtmf, websocket
+Slug: hackathon-2022-building-a-pong
+Status: published
+---
```

Comment:
Document header have syntax error. Remove empty line at the begging and add new empty line a the end.